keyboards/handwired/lagrange/config.h

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2020 Dimitris Papavasiliou <dpapavas@protonmail.ch>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x2718
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Dimitris Papavasiliou
+#define PRODUCT         Lagrange
+#define DESCRIPTION     An ergonomic, split keyboard with thumb clusters.
```

Comment:
This is no longer used. 
```suggestion
```

Reply:
Ah.  I saw that in the PR checklist, but interpreted it as "don't use an empty `#define  DESCRIPTION`" (i.e. defining it to nothing), as all existing keyboards seem to have the define.  I'll remove it.